content/docs/intro/concepts/organizing-stacks-projects.md

```diff
@@ -197,9 +195,21 @@ service = core.v1.Service(..., ResourceOptions(provider=provider))
 {{% choosable language go %}}
 
 ```go
-// StackReference is not supported in Go currently.
-//
-// See https://github.com/pulumi/pulumi/issues/1614.
+import (
+	"fmt"
+
+	"github.com/pulumi/pulumi/sdk/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+    org := cfg.Require("org")
```

Comment:
Where does cfg come from?  Also is indentation off here?

Reply:
Oops, meant to remove the reference to config.

Reply:
Yeah, I think this is a tabs vs spaces thing. It only shows up in the diff, must be due to my editor settings. The rendering on the page looks fine, but I fixed it for cleanliness.